src/app/features/group-list/group-item/group-item.component.scss

```diff
@@ -0,0 +1,28 @@
+.group-item-container {
+    padding: 8px 16px;
+    position: relative;
+}
+
+.group-item {
+    border: 1px solid black;
+    border-radius: 20px;
+    background-color: #F5F5F5;
```

Comment:
czy to jest kolor z theme? jeśli tak użyjmy ze zmiennej, jeśli nie - dodajmy do theme lub użyjmy koloru zdefiniowanego w theme aplikacji

Reply:
Pobrane kolory z paletty